packages/flutter/lib/src/material/tabs.dart

```diff
@@ -2328,6 +2338,8 @@ class _TabsPrimaryDefaultsM3 extends TabBarTheme {
   TabAlignment? get tabAlignment => isScrollable ? TabAlignment.startOffset : TabAlignment.fill;
 
   static double indicatorWeight = 3.0;
+
+  static const EdgeInsetsGeometry iconMargin = EdgeInsets.only(bottom: 2);
```

Comment:
Does this also update `TabBar.secondary`?

Reply:
Yes it does, but the Tab (which is the one applying this margin), is not aware of primary/secondary. Only TabBar knows that.
I was doubting between adding it static to PrimaryDefaults or create a global const _tabIconMarginM3 with the template. But since you wrote Primary in the code snippet above I went with the first approach

Reply:
How do you feel about the static field in this class which applies to both? Or do you have another suggestion?, considering that the Tab widget is not aware in which type of TabBar is positioned.

Reply:
I think `_TabsPrimaryDefaultsM3` static field is fine. This is just a temporary method (other similar classes do this too), eventually it will be replaced when we get update tokens in Flutter.